tst/EnergyPlus/unit/SurfaceGeometry.unit.cc

```diff
@@ -166,3 +166,67 @@ TEST_F( EnergyPlusFixture, BaseSurfaceRectangularTest )
 	EXPECT_FALSE( ErrorsFound );
 	EXPECT_EQ( Polygonal, Surface( ThisSurf ).Shape );
 }
+
+TEST_F( EnergyPlusFixture, ConfirmCheckSubSurfAzTiltNorm )
+{
+	SurfaceData BaseSurface;
+	SurfaceData SubSurface;
+	bool surfaceError;
+
+	//Case 1 - Base surface and subsurface face the same way - should be no error message and no surfaceError
+	surfaceError = false;
+	BaseSurface.Azimuth = 0.;
+	BaseSurface.Tilt = 0.;
+	BaseSurface.NewellSurfaceNormalVector.x = 0.;
+	BaseSurface.NewellSurfaceNormalVector.y = 0.;
+	BaseSurface.NewellSurfaceNormalVector.z = 1.;
+
+	SubSurface.Azimuth = 0.;
+	SubSurface.Tilt = 0.;
+	SubSurface.NewellSurfaceNormalVector.x = 0.;
+	SubSurface.NewellSurfaceNormalVector.y = 0.;
+	SubSurface.NewellSurfaceNormalVector.z = 1.;
+	checkSubSurfAzTiltNorm( BaseSurface, SubSurface, surfaceError );
+	EXPECT_FALSE( surfaceError );
+	EXPECT_TRUE( compare_err_stream( "" , true ) );
+
+	//Case 2 - Base surface and subsurface face the opposite way - should be error message and surfaceError=true
+	surfaceError = false;
+	SubSurface.Azimuth = 180.;
+	SubSurface.Tilt = 180.;
+	SubSurface.NewellSurfaceNormalVector.x = 1.;
+	SubSurface.NewellSurfaceNormalVector.y = 0.;
+	SubSurface.NewellSurfaceNormalVector.z = 0.;
+	checkSubSurfAzTiltNorm( BaseSurface, SubSurface, surfaceError );
+	EXPECT_TRUE( surfaceError );
+	//EXPECT_FALSE( compare_err_stream( "" , true ) );
```

Comment:
@mjwitte Nevermind about the seg fault issue, I had `ENABLE_GTEST_DEBUG_MODE` on when I usually do not.

If you expect those severe errors, why not put the error message in the `compare_err_stream()` call instead of ""? That'll do double duty then for testing the error message output itself.